boot/bootutil/include/bootutil/crypto/sha256.h

```diff
@@ -27,6 +28,9 @@
 
 #if defined(MCUBOOT_USE_MBED_TLS)
     #include <mbedtls/sha256.h>
+#if defined (MBED_TLS_VERSION_3_0_0)
```

Comment:
Given that these are indented, how about indenting the `#if defined` lines added. e.g.
```
#if defined(MCUBOOT_USE_MBED_TLS)
    #include <mbedtls/sha256.h>
    #if defined(MBED_TLS_VERSION_3_0_0)
        #include <mbedtls/compat-2.x.h>
    #endif
    ....
#endif
```

Reply:
done